python/ray/tune/logger/aim.py

```diff
@@ -0,0 +1,209 @@
+import logging
+import numpy as np
+
+from typing import TYPE_CHECKING, Dict, Optional
+
+from aim.ext.resource import DEFAULT_SYSTEM_TRACKING_INT
```

Comment:
I think we should delay the import here -- i.e., if aim is not installed, we should not fail